### ISSUE TITLE
My Veikk S640 is detected instantly but my Veikk a15 pro does not

### ISSUE THREAD

**Issue by DrunkenMasterTim** [OpenTabletDriver/OpenTabletDriver#2136]
I have tried everything uninstalled drivers reinstalled countless times reinstalled framework, my a15 pro will not detect at all but my s640 connects instantly everytime

**Comment by jamesbt365** [comment]
Help > Export diagnosis, save that file as a `.txt` and attach it here

**Comment by DrunkenMasterTim** [comment]
[diagnosis.txt](https://github.com/OpenTabletDriver/OpenTabletDriver/files/8356821/diagnosis.txt)

**Comment by jamesbt365** [comment]
```    {
      "Time": "2022-03-27T00:38:30.0235497-04:00",
      "Group": "Detect",
      "Message": "Another tablet driver found: Veikk",
      "StackTrace": null,
      "Level": 2,
      "Notification": false
    },
    {
      "Time": "2022-03-27T00:38:30.0235878-04:00",
      "Group": "Detect",
      "Message": "Detection for Veikk tablets might be impaired",
      "StackTrace": null,
      "Level": 2,
      "Notification": false
    },
```
uninstall then export diagnostic again.

**Comment by DrunkenMasterTim** [comment]
[New Text Document.txt](https://github.com/OpenTabletDriver/OpenTabletDriver/files/8358423/New.Text.Document.txt)

**Comment by DrunkenMasterTim** [comment]
[New Text Document.txt](https://github.com/OpenTabletDriver/OpenTabletDriver/files/8363424/New.Text.Document.txt)
use this one didnt have tablet plugged in for the last one

**Comment by jamesbt365** [comment]
Create a `Configurations` folder by the executables of OpenTabletDriver and move [this](https://github.com/OpenTabletDriver/OpenTabletDriver/files/8402537/A15.Pro.json.txt) here, then remove the `.txt` at the of the file teh name leaving it as `A15 Pro.json`

**Comment by DrunkenMasterTim** [comment]
i just did it, its detected but now when i hover over the tablet the cursor just goes crazy up and down

**Comment by jamesbt365** [comment]
fixed in #2212
